platform_versioned_docs/version-24.1.0/enterprise/release_notes/enterprise_latest.mdx

```diff
@@ -4,18 +4,15 @@ Seqera Platform Enterprise version 24.1.0 introduces two new features: Data Expl
 
 ## Seqera Compute
 
-Seqera Compute is a fully optimized runtime designed to revolutionize scientific computing on Seqera Platform. Now available in private preview, Seqera Compute enables scientists to concentrate on their work by integrating compute, storage, credential management, and performance optimizations into a single, cohesive service. 
+Seqera Compute is a fully optimized runtime designed to revolutionize scientific computing on Seqera Platform. Now available in private preview, Seqera Compute enables scientists to concentrate on their work by integrating compute, storage, credential management, and performance optimizations into a single, cohesive service.
 
 ## Data Studios
 
-Data Studios closes the loop from development, to deployment and insights, allowing you to create, manage, share notebook environments in Seqera with the click of a button. It's now easier than ever to transition Nextflow-generated data to JupyterLab, RStudio, and VSCode environments with pre-built templates that can easily leverage existing compute environments and data in Seqera Platform.
-
-Data Studios also makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work that enable collaboration and reproducibility while remaining secure.
-Get inspired by examples running Data Studios, in the Seqera Platform Community Showcase. 
+[Data Studios](./data-studios.mdx) closes the loop from development, to deployment and insights, allowing you to create, manage, share notebook environments in Seqera with the click of a button. It's now easier than ever to transition Nextflow-generated data to JupyterLab, RStudio, and VSCode environments with pre-built templates that can easily leverage existing compute environments and data in Seqera Platform. Data Studios also makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work that enable collaboration and reproducibility while remaining secure.
 
 ## Data Explorer
 
-Data Explorer is now Generally Available and from version 24.1 it supports multi-file and multi-folder download. 
+[Data Explorer](./data-explorer.mdx) is now Generally Available and from version 24.1 it supports multi-file and multi-folder download.
 
 This allows users to bypass the need for cloud credentials or complex DevOps configurations and easily download entire datasets of pipeline results with to share with their team. This is a user-friendly alternative to traditional cloud storage services without the hassle of a command-line interface or intricate user interfaces.
```

Comment:
I think you mean "bypass user-owned/managed credentials" since they can leverage the credentials present in the Workspace? Maybe call out the ability to interact with a variety of buckets without having to leave Seqera Platform rather than having to open other browser windows/tooling?